hbase-server/src/main/java/org/apache/hadoop/hbase/tool/BulkLoadHFiles.java

```diff
@@ -84,6 +84,10 @@ public Path getFilePath() {
   Map<LoadQueueItem, ByteBuffer> bulkLoad(TableName tableName, Map<byte[], List<Path>> family2Files)
       throws TableNotFoundException, IOException;
 
+  /**
+   * Disable replication for this bulkload, if bulkload replication is configured.
```

Comment:
What about: `Disables replication for all bulkloads done via this instance, when bulkload replication is configured.`

"this bulkload" could be interpreted as something the user has to call before each call to `bulkLoad(..)`.